x/incentives/keeper/distribute_test.go

```diff
@@ -2080,3 +2099,121 @@ func (s *KeeperTestSuite) setupVolumes(poolIds []uint64, updatedPoolVolumes []os
 		s.App.PoolManagerKeeper.SetVolume(s.Ctx, poolIds[i], sdk.NewCoins(sdk.NewCoin(s.App.StakingKeeper.BondDenom(s.Ctx), updatedVolume)))
 	}
 }
+
+func (s *KeeperTestSuite) TestHandleGroupPostDistribute() {
+	tests := map[string]struct {
+		groupGauge       types.Gauge
+		coinsDistributed sdk.Coins
+		amountToPrefund  sdk.Coins
+		// boolean because there is only one error that we can setup in tests.
+		expectError bool
+	}{
+		"1: perpetual gauge - updated": {
+			groupGauge:       defaultPerpetualGauge,
+			coinsDistributed: defaultCoins,
+		},
+		"2: non-perpetual gauge - updated": {
+			groupGauge:       withNonPerpetualEpochs(withIsPerpetual(deepCopyGauge(defaultPerpetualGauge), false), 0, 2),
+			coinsDistributed: defaultCoins,
+		},
+		"3: non-perpetual gauge - deleted": {
+			groupGauge:       withNonPerpetualEpochs(withIsPerpetual(deepCopyGauge(defaultPerpetualGauge), false), 0, 1),
+			coinsDistributed: defaultCoins,
+		},
+
+		"4: non-perpetual gauge - deleted (non-zero community pool update)": {
+			// Note that total coins in gauge = 2x defaultCoins
+			// Already distributed coins = none
+			// Coins to distribute = defaultCoins
+			// Therefore, the community pool should receive 2x defaultCoins - defaultCoins = defaultCoins
+			groupGauge:       withCoinsToDistribute(withNonPerpetualEpochs(withIsPerpetual(deepCopyGauge(defaultPerpetualGauge), false), 0, 1), defaultCoins.Add(defaultCoins...), sdk.NewCoins()),
+			coinsDistributed: defaultCoins,
+			amountToPrefund:  defaultCoins,
+		},
+
+		// // edge cases
```

Comment:
Assuming this was intentionally double-commented to make it clear it's a separator? Might be good for us to align on a separator pattern that we all like for consistency (I'm personally indifferent as long as it's roughly consistent across our codebase)

Reply:
This was a typo. Addressed. Happy to make an issue to discuss comment consistency in tests